RadarSDK/RadarLocationManager.m

```diff
@@ -165,17 +166,18 @@ - (void)startTrackingWithOptions:(RadarTrackingOptions *)trackingOptions {
     CLAuthorizationStatus authorizationStatus = [self.permissionsHelper locationAuthorizationStatus];
     if (!(authorizationStatus == kCLAuthorizationStatusAuthorizedWhenInUse || authorizationStatus == kCLAuthorizationStatusAuthorizedAlways)) {
         [[RadarDelegateHolder sharedInstance] didFailWithStatus:RadarStatusErrorPermissions];
-
         return;
     }
 
     [RadarSettings setTracking:YES];
     [RadarSettings setTrackingOptions:trackingOptions];
+    [RadarSettings setFallbackTrackingOptions:trackingOptions];
     [self updateTracking];
 }
 
 - (void)stopTracking {
     [RadarSettings setTracking:NO];
+    [RadarSettings setListenToServerTrackingOptions:NO];
```

Comment:
will this ever get sent to the server? wondering if this will actually decrease the number of clients using remote tracking options in our system

Reply:
yes, it doesn't have any functional behaviour apart from analytics